content/docs/operating/configuration.md

```diff
@@ -6,66 +6,189 @@ nav_icon: sliders
+
+```
+# The job name assigned to scraped metrics by default.
+job_name: <name>
+
+# How frequently to scrape targets from this job.
+[ scrape_interval: <duration> | default = <global_config.scrape_interval> ] 
+
+# Per-target timeout when scraping this job.
+[ scrape_timeout: <duration> | default = <global_config.scrape_timeout> ]
+
+# The HTTP resource path on which to fetch metrics from targets.
+[ metrics_path: <path> | default = /metrics ]
+
+# The URL scheme with which to fetch metrics from targets.
+[ scheme: <scheme> | default = http ] 
+
+# List of DNS service discovery configurations.
+dns_configs:
+  [ - <dns_config> ... ]
+
+# List of labeled target groups for this job.
+target_groups:
+  [ - <target_group> ... ]
+
+# List of relabel configurations.
+relabel_configs:
+  [ - <relabel_config> ... ]
+```
+
+Where `<scheme>` may be `http` or `https` and `<path>` is a valid URL path.
+`<job_name>` must be unique across all scrape configurations and adhere to the
+regex `[a-zA-Z_][a-zA-Z0-9_-]`.
+
+
+### Target groups `<target_group>`
+
+Target Groups collect a group of targets and specify a common label set for them.
+They are the canoncial way to specify static targets in a scrape config.
+
+```
+# The targets specified by the target group.
+targets: 
+  [ - '<host>' ]
+
+# Labels assigned to all metrics scraped from the targets.
+labels:
+  [ <labelname>: <labelvalue> ... ]
+```
+
+Where `<host>` is a valid string consisting of a hostname or IP followed by a port 
+number.
+
+
+### DNS configurations `<dns_config>`
+
+A DNS configuration allows to specify a set of hosts for which DNS SRV records a
+queried. The DNS servers to be contacted are read from `/etc/resolv.conf`.
+
+The label `__meta_dns_srv_name` is attached to discovered targets with the queried
+SRV name as its value.
+
+```
+# A list of host names to be queried.
+names:
+  [ - <host> ]
+
+# The time after which the provided names are refreshed.
+[ refresh_interval: <duration> | default = 30s ]
+```
+
+Where `<host>` is a valid hostname.
+
+
+### Relabeling `<relabel_config>`
+
+Relabeling is a powerful tool to dynamically rewrite the label set of a target. Multiple
+relabeling steps can be configured per scrape config. They are applied to the label set of
+each target in order of their configuration.
+
+Initially, aside from the configured labels, the `job` label is set to the `job_name` value
+of the surrounding scrape configuration. The `__address__` label is set to the `<host>:<port>`
+value of the target.
+After relabeling the `instance` label is set to the value of `__address__` by default if
+it was not set during relabeling.
 
-Below is an example configuration which explains the most common options with
-some comments:
+Additional labels prefixed with `__meta_` may be available for relabeling. They are set
+by the service discovery mechanism that provided the target.
+
+Labels starting with `_` will be removed from the label set after relabeling is completed.
```

Comment:
Mh, yes, might change that. In general manual deletion is still possible by setting an empty value.